testing/setup.py

```diff
@@ -12,5 +12,5 @@
         'docker>=3.0.0',
         'signalfx>=1.0',
     ],
-    python_requires='>=3.5',
+    python_requires='>=2.6',
```

Comment:
This is contradictory since we require docker>=3.0.0 which does not support 2.6.  There is also no support in python for the concept of certain modules in a package being compatible with certain python versions. This puts hidden constraints on this package about how it can import code that is incompatible with py 2.6 and how modules within the package can interact with each other which will make it more complex to maintain.

I think the main issue is that this package is for collectd _integration testing_ and not unit testing.  That is by far the most valuable form of testing for collectd plugins.